specs/chef/run_cookbook_manually.yml

```diff
@@ -0,0 +1,15 @@
+---
+name: Manually run a cookbook by name
+command: sudo chef client -o recipe['{{cookbook}}'] 
+tags:
+  - macos
+  - linux
```

Comment:
Mind removing the `macos` and `linux` tags here? From our experience, the more specific tags to the workflow, the better

Reply:
no problem